cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/EvaluationOrderGraphPass.kt

```diff
@@ -599,6 +600,7 @@ open class EvaluationOrderGraphPass(ctx: TranslationContext) : TranslationUnitPa
 
         createEOG(node.tryBlock)
         val tmpEOGNodes = ArrayList(currentPredecessors)
+        val catchEnds = ArrayList<Node>()
```

Comment:
minor: should be created liek this: 
```suggestion
        val catchEnds = mutableListOf<Node>()
```

same for the line above